app/services/preference_set_table_state_service.rb

```diff
@@ -12,7 +12,7 @@ def update!(table_name:, table_state:)
     preference_set.save!
 
     unless preference_set.save
-      raise UpdateFailedError, "Failed to update table state for table_name: '#{table_name}'"
+      raise "Failed to update table state for table_name: '#{table_name}'"
     end
```

Comment:
to reclarify, these lines are useless and will never be triggered due to the way `#save!` works.